vike/node/runtime/renderPage.ts

```diff
@@ -149,6 +149,14 @@ async function renderPageAndPrepare(
     // From now on, renderContext.pageConfigs contains all the configuration data; getVikeConfig() isn't called anymore for this request
   }
 
+  if (renderContext.onBootHook) {
+    const onBootResult = await renderContext.onBootHook?.hookFn(pageContextInit) as Record<string, unknown>
```

Comment:
I don't think we should call `onBoot()` at request-time? My thinking was to call `onBoot()` exactly once and as soon as possible whenever its code is loaded and then cache its result.